impl/src/main/java/org/jboss/cdi/tck/interceptors/tests/contract/aroundConstruct/Charlie.java

```diff
@@ -18,7 +18,8 @@
 
 import jakarta.interceptor.Interceptors;
 
-@Interceptors({CharlieInterceptor1.class, CharlieInterceptor2.class})
+//@Interceptors({CharlieInterceptor1.class, CharlieInterceptor2.class})
```

Comment:
I think I'd just remove the `@Interceptors` annotations altogether, but commenting them out is probably fine too -- especially if you want to copy the tests later to a `full` package to also have coverage for `@Interceptors`.

Reply:
I'd just remove this. If we want a copy of tests for `@Interceptors` we should create them now while we know we need them :)